README.md

```diff
@@ -156,6 +156,17 @@ which fields the ordering is enabled for, pass an array of the field names into
 />
 ```
 
+To have the opposite effect simply use `excludeOrderingBy`:
+
+```JSX
+<AjaxDynamicDataTable
+    rows={this.state.users}
+    excludeOrderingBy={[
+        'dob'
+    ]}
+/>
+```
+
```

Comment:
```suggestion

Typically, the `allowOrderingBy` and `excludedOrderingBy` props should not be used simultaneously, as this could cause unexpected behaviour.

```